nifi-api/src/main/java/org/apache/nifi/components/AbstractConfigurableComponent.java

```diff
@@ -126,19 +125,6 @@ private PropertyDescriptor getSupportedPropertyDescriptor(final PropertyDescript
             }
         }
 
-        // validate any dynamic properties
-        for (final Map.Entry<PropertyDescriptor, String> entry : context.getProperties().entrySet()) {
-            final PropertyDescriptor descriptor = entry.getKey();
-            final String value = entry.getValue();
-
-            if (supportedDescriptors != null && !supportedDescriptors.contains(descriptor)) {
```

Comment:
I think the `supportedDescriptors` is no longer needed (line 96)

Reply:
Oops! Good catch. Removed supportedDescriptors.